include/git2/remote.h

```diff
@@ -137,7 +137,8 @@ GIT_EXTERN(git_repository *) git_remote_owner(const git_remote *remote);
 GIT_EXTERN(const char *) git_remote_name(const git_remote *remote);
 
 /**
- * Get the remote's url
+ * Get the remote's url. If url.*.insteadOf has been configured
+ * for this URL, it will return the modified URL.
```

Comment:
The clarification about the insteadof rules should be in a different paragraph. Same goes for the one below.